translate/src/modules/entitydetails/components/Metadata.tsx

```diff
@@ -86,6 +86,36 @@ function GroupComment({ comment }: { comment: string }) {
   );
 }
 
+function EntityCreatedDate({ dateCreated }: { dateCreated: string }) {
+  // Create date and time formatters
+  const dateFormatter = new Intl.DateTimeFormat('en-US', {
+    day: 'numeric',
+    month: 'long',
+    year: 'numeric',
+  });
+  const timeFormatter = new Intl.DateTimeFormat('en-US', {
+    hour: '2-digit',
+    minute: '2-digit',
+    second: '2-digit',
+  });
+
+  // Create a Date object from the dateCreated string
+  const date = new Date(dateCreated);
+
+  // Format the date and time
+  const formattedDate = dateFormatter.format(date);
+  const formattedTime = timeFormatter.format(date);
+
+  // Combine the formatted date and time into one string
+  const formattedDateTime = `${formattedDate} ${formattedTime}`;
+
+  return (
+    <Datum id='entity-created-date' title='CREATED DATE'>
```

Comment:
Please make sure you add `entitydetails-Metadata--entity-created-date` item to the [`translate.ftl`](https://github.com/mozilla/pontoon/blob/master/translate/public/locale/en-US/translate.ftl#L273) file. I would just use CREATED or CREATED ON, because it also includes time.